doc-content/enterprise-only/project-data/creating-change-requests-proc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='creating-change-requests-proc_{context}']
+
+= Creating change requests
+You can create a change request in a {CENTRAL} project after you have made a change in your project, for example after you have added or deleted an attribute to an asset.
+
+.Prerequisites
+* You have more that one branch of a {CENTRAL} project.
+* You made a change in one branch that you want to merge to another branch.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and select the space and project that contains the change that you want to merge.
+. On the project page, select the branch that contains the change.
++
+.Select a branch menu
+image::getting-started/change-branch.png[Selecting a branch menu]
```

Comment:
This image in the draft does not seem updated since I cannot see the _Change Requests_ tab.

Reply:
Image updated